include/swift/Sema/CSFix.h

```diff
@@ -1178,18 +1178,21 @@ class AllowClosureParamDestructuring final : public ConstraintFix {
                                                 ConstraintLocator *locator);
 };
 
+struct SynthesizedArg {
+  unsigned paramIdx;
+  AnyFunctionType::Param param;
+};
+
```

Comment:
This is no longer necessary (in my original changes it also tracked the index to insert the missing arg at) but I left it in for the slight readability improvement over a `std::pair` (e.g. `paramIdx` vs `first`)

Reply:
That works for me, thank you!